### PR TITLE
Wallet details

### PR DESCRIPTION
**Pull request by JDonadio**

### CODE REVIEW

File: src/pipes/toFiat.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+@Pipe({ name: 'toFiat' })
+export class ToFiatPipe implements PipeTransform {
+  private unitCode: string;
+
+  constructor(
+    private configProvider: ConfigProvider,
+    private txFormatProvider: TxFormatProvider
+  ) { 
+    this.unitCode = this.configProvider.get()['wallet']['settings'].unitCode;
```
Comment (cmgustavo) [inline src/pipes/toFiat.ts]: configProvider.get() return a typed object. Not necessary to use ['xx'].

File: src/app/app.module.ts [hunk 2]
```diff
@@ -186,8 +193,13 @@ let providers: any = [
   }
 ];
 
+let pipes = [
```
Comment (cmgustavo) [inline src/app/app.module.ts]: +1

File: src/pipes/toFiat.ts (hunk 1 above)
Comment (JDonadio) [reply src/pipes/toFiat.ts]: Got it!

File: src/pages/home/home.ts [hunk 3]
```diff
@@ -64,4 +65,8 @@ export class HomePage {
   public goToAddView(): void {
     this.navCtrl.push(AddPage);
   }
+
+  goToWalletDetails(wallet: any) {
+    this.navCtrl.push(WalletDetailsPage, {wallet: wallet});
```
Comment (cmgustavo) [inline src/pages/home/home.ts]: `wallet` param should be replaced by a single element, not an object. Ej, `walletId`

Comment (JDonadio) [reply src/pages/home/home.ts]: +1

File: src/pages/tx-details/tx-details.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+  private wallet: any;
+  
+  constructor(
+    private navParams: NavParams,
+    private txFormatProvider: TxFormatProvider,
+    private walletProvider: WalletProvider,
+    private externalLinkProvider: ExternalLinkProvider,
+  ) {
+    this.wallet = this.navParams.data.wallet;
```
Comment (cmgustavo) [inline src/pages/tx-details/tx-details.ts]: Same as above. Only receive `walletId`, then this.profileProvider.getWallet(id)

Comment (JDonadio) [reply src/pages/tx-details/tx-details.ts]: +1

**Comment by JDonadio** [comment]
Updated

**Comment by 270815** [comment]
https://github.com/270815/wallet.git

File: src/pages/tx-details/tx-details.ts (hunk 4 above)
Comment (270815) [reply src/pages/tx-details/tx-details.ts]: #82e71c1

**Comment by gabrielbazan7** [comment]
ACK

**Pull request merged**